src/compile/render-ssr/handlers/Element.ts

```diff
@@ -2,8 +2,9 @@ import { quotePropIfNecessary, quoteNameIfNecessary } from '../../../utils/quote
 import isVoidElementName from '../../../utils/isVoidElementName';
 import Attribute from '../../nodes/Attribute';
 import Node from '../../nodes/shared/Node';
-import { escape, escapeTemplate } from '../../../utils/stringify';
-import { snip } from '../utils';
+import { snip } from '../../../utils/snip';
+import { stringify_attribute } from '../../../utils/stringify_attribute';
```

Comment:
The distinction is between code that was written before a certain date and code that was written after it... a while back I decided that snake case almost always makes more sense than camel case and just started using it everywhere. Will probably do a sweep and update all the came case stuff soon (except public APIs — the JS community probably isn't ready to have that particular opinion forced upon them) because you're right, it's a bit mysterious